runtime/src/lib.rs

```diff
@@ -754,43 +810,48 @@ pub type XcmOriginToTransactDispatchOrigin = (
 );
 
 parameter_types! {
-	// One XCM operation is 1_000_000 weight - almost certainly a conservative estimate.
-	pub UnitWeightCost: Weight = 1_000_000;
-	// One UNIT buys 1 second of weight.
-	pub const WeightPrice: (MultiLocation, u128) = (MultiLocation::parent(), UNIT);
+	// One XCM operation is 1_000_000_000 weight - almost certainly a conservative estimate.
+	pub UnitWeightCost: Weight = 1_000_000_000;
+	pub const MaxInstructions: u32 = 100;
```

Comment:
why do we have 100 max instructions?

Reply:
from here https://github.com/paritytech/polkadot/blob/v0.9.13/runtime/polkadot/src/lib.rs#L1348-L1350